py-scripts/lf_base_interop_profile.py

```diff
@@ -1491,16 +2000,78 @@ def main():
     if (args.config_wifi):
         real_devices = RealDevice(manager_ip=args.host,
```

Comment:
These initializers are long and somewhat difficult to read. No change required here, but consider for future use `RealDevice(**vars(args))` approach, as is present in some of the creation scripts, like `create_station.py`.